services/app-web/src/components/Form/FieldYesNo/FieldYesNo.tsx

```diff
@@ -23,7 +23,7 @@ export type FieldYesNoProps = {
     hint?: React.ReactNode
     showError?: boolean
     id: string
-    variant?: 'PRIMARY' | 'SECONDARY' // secondary variant used for nested fields yes/no fields under an overarching heading
+    variant?: 'TOPLEVEL' | 'SUBHEAD' // subhead variant used for nested fields yes/no fields under an overarching heading
```

Comment:
@macrael FYI saw your comment after I merged but... your wish is granted 🪄 https://github.com/CMSgov/managed-care-review/pull/1390#discussion_r1053541129